Add entry for flag documentation (#4403)
knative/docs: docs/help/contributor/style-guide/documenting-code.md

    <some-code>
    ```
    ````

## Documentation of flags

> > Always use the short version of flags used in a command than the extended version, i.e., use ```-n``` rather than ```--namespace```

Reviewer: ```suggestion
> Always use the short version of flags used in a command rather than the extended version. For example, use `-n` rather than `--namespace`.
```
